### PR TITLE
Update the css-images IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**
Hello reviewer(s),

This PR is intended to consolidate the spec’s IDL definition with the WPT test suite’s copy, and any idlharness tests.

The up-to-date copy of the IDL file was automatically extracted from the reffy-reports repo (https://github.com/tidoust/reffy-reports/tree/master/whatwg/idl) which scrapes known specs automatically + regulary.

This PR is part of a migration project which will eventually be automatically updating and creating PRs for changes in spec IDL.

Please check that:
The spec (and its source) is correct and up-to-date
All tests which cover the IDL in the spec have been migrated to fetch + use the idl in the `interfaces/` directory (instead of inline copies in the test files).

### CODE REVIEW

**Comment by lukebjerring** [comment]
Blocked by https://github.com/web-platform-tests/wpt/issues/7583

File: interfaces/css-images.idl
Comment (foolip) [inline interfaces/css-images.idl]: Namespaces now supported, but this will have to change after or together with an update of cssom.idl

File: css/css-images/idlharness.html
Comment (foolip) [inline css/css-images/idlharness.html]: Should depend on cssom

Comment (lukebjerring) [reply css/css-images/idlharness.html]: Done.

File: interfaces/css-images.idl
Comment (lukebjerring) [reply interfaces/css-images.idl]: The test actually currently passes.. so it's clearly not asserting that it's a namespace. Will send you a PR.

Comment (foolip) [reply interfaces/css-images.idl]: Filed after I reviewed this: https://github.com/web-platform-tests/wpt/issues/12123

**Review by foolip — APPROVED** [review]
This LGTM as long as namespaces don't blow up.

**Comment by lukebjerring** [comment]
Merging as-is. `cssom` will be updated soon via #11819

**Pull request merged**
